apiml-tomcat-common/src/main/java/org/zowe/apiml/product/web/ApimlTomcatCustomizer.java

```diff
@@ -85,7 +86,13 @@ public SocketState process(SocketWrapperBase<S> socket, SocketEvent status) {
             } catch (ClassNotFoundException | NoSuchFieldException | IllegalAccessException e) {
                 throw new AttlsHandlerException("Different implementation expected.", e);
             } finally {
-                InboundAttls.dispose();
+                try {
+                    InboundAttls.clean();
+                } catch (ContextIsNotInitializedException e) {
+                    log.debug("Cannot clean AT-TLS context: {}", e.getMessage());
```

Comment:
is it possible for a message to be empty? Maybe we can print some more info here

Reply:
You are right, the message is always empty. Anyway, the exception there is just theoretical, and we do not have any meaningful information to show in addition. So I removed the message from the log entry.